### PR TITLE
Building Containers for Scientific Workflows blog post

### PR DESCRIPTION
**Pull request by mribeirodantas**

### CODE REVIEW

**Comment by pditommaso** [comment]
Is this ready to go?

**Comment by mribeirodantas** [comment]
On my side, yes, but I'm not sure if @evanfloden had a look at it yet. Besides, it would be nice to know when it'll be out so I can set properly the `date` field.

**Comment by pditommaso** [comment]
then, better mark as "Draft"

**Comment by mribeirodantas** [comment]
Thanks!

File: content/blog/2022/building-containers-for-scientific-workflows.md [hunk 1]
```diff
@@ -0,0 +1,493 @@
+title= Building Containers for Scientific Workflows
+date=2022-08-28
```
Comment (mribeirodantas) [inline content/blog/2022/building-containers-for-scientific-workflows.md:2]:
```suggestion
date=2022-10-20
```

**Comment by mribeirodantas** [comment]
It's been already published.

**Pull request closed without merging**